src/views/migrationpolicies/list/utils/constants.ts

```diff
@@ -1,3 +1,10 @@
 import { MigrationPolicyModelRef } from '@kubevirt-ui/kubevirt-api/console';
 
 export const migrationPoliciesPageBaseURL = `/k8s/cluster/${MigrationPolicyModelRef}`;
+
+// t('With form')
```

Comment:
you can import t and avoid using comments for translation

Reply:
I am confused. Some people prefer this, some that.. We should decide what is preferred form to use t function in our codebase and we all should follow it.

Reply:
It's not preference. At the begining this was the only option using comment. Now for not so short time we are using t. Give us more control on our code. U already used it sever times in utils files I'm guessing this is no exception.

Reply:
https://github.com/kubevirt-ui/kubevirt-plugin/pull/901 - nothing new :)

Reply:
Well, I often experience very very different opinions on one topic, depending on who reviews the PR.. Sometimes it is just impossible to make everybody happy.